docs/database/readme.md

```diff
@@ -0,0 +1,159 @@
+# Schema
+
+```mermaid
+---
+title: Airneis database model
+---
+
+classDiagram
+
+Categories <|-- Products
+Carts <|-- Status
+Products <|-- Carts
+Users <|-- Addresses
+Users <|-- Carts
+Carts <|-- CartsProducts
+Products <|-- CartsProducts
+Products <|-- ProductImages
+Materials <|-- ProductsMaterials
+Products <|-- ProductsMaterials
+
+class Users {
+    id increments notNull
+    fullName text notNull
+    email text unique notNull
+    isAdmin boolean default false notNull
+    isActive boolean default false notNull
+}
```

Comment:
Can you add the login fields, passwordHash, Salt...etc

Reply:
Why can't the email be the loggin?

Reply:
The email is perfect for the login but we have some missing fields if not mistaken